### ISSUE TITLE
When mason fields are in Banner, DiscussionPage-nav not showing properly

### ISSUE THREAD

**Issue by prestorondo** [flagrow/mason#5]
If you choose "Show fields in the discussion banner instead of the "first post" , in any discussion page, the right navigator will not render properly .(~~because of it's `<ul>` has `position:fixed` , a quick change to `position:absolute` would fix the problem~~* )

I do not know whether this issue is related to `flarum/core` or this extension.

* _I realised shortly after I changed that the behaviour changes (the nav isn't sticky anymore which is not cool 😢  ) if we do this, please ignore this fix._

**Comment by luceos** [comment]
Can you post a screenshot perhaps?

**Comment by prestorondo** [comment]
![mason-hero](https://user-images.githubusercontent.com/9180251/39759972-5914ee56-52d4-11e8-9ddb-dfd6515cbcf1.gif)

if I am on my big monitor, I do not have that problem but on tablet I still see it.

**Comment by luceos** [comment]
I'm in doubt, I'd say this is a core issue. What do you think @clarkwinkelmann ?

**Comment by clarkwinkelmann** [comment]
Yes I'm also blaming core :joy: The sidebar simply stays where it was at page load. Which is fine with the default design, less fine when you add stuff to the header.

Not sure how easy to fix it is (in core) though. Depends what library is responsible for that affix effect and whether there's an option to more it higher when scrolling...
